NLP/BERT_PyTorch/readme.md

```diff
@@ -0,0 +1,95 @@
+# PyTorch Implementation of BERT
+
+## Usage
+
+### For Pretraining and Classification task in Colab
+
+```python
+!pip install transformer  #(for pretrained weights)
+!pip install pytorch_pretrained_bert  #(for BERT vocab)
+!pip install wget
+!git clone link-to-repo
+%run pretrain.py --savedir 'path to directory where you want to save checkpoint' --corpus 'path to train corpus'
+%run classify_cola.py --pretrain_file 'path of pretrain weights'
+```
```

Comment:
Convert these to shell commands so that users can use you code on their local machine too. So you can change the title to `For Pretraining and Classification task` after this change.